static/app/views/replays/detail/network/index.tsx

```diff
@@ -169,6 +172,42 @@ function NetworkList({
     );
   };
 
+  const handleClick = useCallback(() => {
+    const frame = getNextReplayFrame({
+      frames: items,
+      targetOffsetMs: currentTime,
+      allowExact: true,
+    });
+    const frameIndex = items.findIndex(spanFrame => frame === spanFrame);
+    // index needs to be at least 1 to be a valid index to jump to
+    const index = frameIndex < 1 ? 1 : frameIndex;
+    setScrollToRow(index);
+  }, [setScrollToRow, currentTime, items]);
+
+  function indexAtCurrentTime() {
+    const frame = getNextReplayFrame({
+      frames: items,
+      targetOffsetMs: currentTime,
+      allowExact: true,
+    });
+    const frameIndex = items.findIndex(spanFrame => frame === spanFrame);
+    // frameIndex is -1 when the page is loading, so the Jump Up Button appears until the page finishes loading in
```

Comment:
I'm not sure if this comment is still true, but i notice that something similar is happening at the bottom of the list.

When I'm near the end of the replay, after all network activity is done, the jump up button is still visible:
<img width="540" alt="SCR-20230818-mpcq" src="https://github.com/getsentry/sentry/assets/187460/4c95f302-37dd-4bf8-8d6f-93b21dd540b0">

We might need different default values for the range or frameIndex depending on if we're checking the scroll-up button or the scroll-down button

Reply:
I made the buttons show up properly at the beginning and end of the timeline, but I still need to update the onClick portion so it jumps to where they're supposed to

Reply:
ok should be all fixed now i hope!